cmd/root.go

```diff
@@ -158,6 +166,17 @@ or ./scorecard --{npm,pypi,rubgems}=<package_name> [--checks=check1,...] [--show
 		switch format {
 		case formatDefault:
 			err = repoResult.AsString(showDetails, *logLevel, os.Stdout)
+		case formatSarif:
+			if !v3 {
+				log.Fatalf("sarif not supported yet")
+			}
+			checkDocs, e := docs.Read()
+			if e != nil {
+				log.Fatalf("cannot read yaml file: %v", err)
+			}
+			// TODO: support config files and update checker.MaxResultScore.
+			// TODO: update version.
+			err = repoResult.AsSARIF("1.2.3", showDetails, *logLevel, os.Stdout, checkDocs, checker.MaxResultScore)
```

Comment:
What does "1.2.3" refer to? sarif version? better to make constant to make it clear what it's referring to.

Reply:
it's the scorecard version. It needs to be a semantic version, something I need to work on. I'll add a variable to make it clearer. Thanks!